submissions/nadsatt/friends-app/modules/filter-group.js

```diff
@@ -0,0 +1,73 @@
+import * as searchFiltersModule from './search-filter-items.js';
+import * as sortFiltersModule from './sort-filter-items.js';
+import { FilterList } from './filter-list.js';
+import { IconList } from './icon-list.js';
+
+export class FilterGroup {
+    constructor(userService, pageLinkList){
+        const categories = ['sort', 'search'];
```

Comment:
```suggestion
        const [sort, search] = ['sort', 'search'];
```

Better then `categories [0]` etc.